src/coreComponents/constitutive/solid/SolidBase.hpp

```diff
@@ -98,7 +98,7 @@ class SolidBaseUpdates
   arrayView1d< real64 const > const m_thermalExpansionCoefficient;
 
   /// Flag to disable inelasticity
-  const bool & m_disableInelasticity;
+  const bool m_disableInelasticity;
```

Comment:
This was a bug on heterogeneous memory systems.

Reply:
is there any way to automatically check for this kind of bugs? Any way to at least get a warning whenever a reference is captured by value in a raja `forAll`?